src/js/disability-benefits/utils/helpers.js

```diff
@@ -199,3 +202,27 @@ export function isClaimComplete(claim) {
 export function itemsNeedingAttentionFromVet(events) {
   return events.filter(event => event.status === 'NEEDED' && event.type === 'still_need_from_you_list').length;
 }
+
+export function makeRequest(url, userOptions = {}) {
```

Comment:
Could this be `makeAuthedRequest` and include the `handleError` function you have as part of the promise chain here? I'm not entirely sure how the `dispatch` function is getting passed around, but it would be nice to not have to include that error handling in each request